src/scripts/charmc

```diff
@@ -1472,7 +1472,12 @@ if [ "$TLSGLOBALS" = "1" ]
 then
     if [[ "$CMK_SUPPORTS_TLSGLOBALS" = '0' ]]
     then
-      echo "Error: -tlsglobals is not supported on $CMK_VDIR"
+	    if [[ "$CMK_USER_DISABLED_TLS" = '1' ]]
+	    then
+        echo "Error: Charm was built with '--disable-tls' and -tlsglobals is not available."
+	    else
+        echo "Error: -tlsglobals is not supported on $CMK_VDIR"
```

Comment:
Inconsistent indentation (the conditional lines have tabs, the statement lines have spaces, it looks like).

Reply:
Changed all to spaces for this range of lines